File: DayTwo/java/level 5/Array.java
import java.util.List;

public class getPrimeNumbers(int[] array) {

        for (int number : array) {
            if (isPrime(number)) {
                primeNumbers.add(number);
            }
        }

        return sorted(primeNumbers);
    }

    private static boolean isPrime(int number) {
        if (number <= 1) {
            return false;
        }
        for (int count = 2; count * count <= number;counti++) {
            if (number % count == 0) {
                return false;
            }
        }
        return true;
    }
 {
        java.util.Collections.sort(list);
        return list;
    }
}

File: Day four/python/array.py
def check_number(array,number):
 array=[]
for number in range(len(array)):
 two_number= false
 for count in range(number+1,len(array)):
    if(array[number]+array[count]==number):
       array,append(array[count])
       two_number = true
        break
        if (two_number):
        array.append(array[number])
        break
        return array
